### ISSUE TITLE
The plot to show the training and validation processes

### ISSUE THREAD

**Issue by quangchien99** [ptran1203/traffic_sign_detection#3]
Can you show me the code the visualize the result after training like the loss, val_loss or accuracy?

**Comment by ptran1203** [comment]
Hi, currently the code for loss curve visualization is unavailable. The metric for this problem is [mAP](https://stackoverflow.com/questions/36274638/what-is-the-map-metric-and-how-is-it-calculated) but I did not implement it yet.

For loss curve visualization you can check the following:

```python
H = model.fit(...)
loss_values = H.history["loss"]
plt.plot(epochs, loss_values, label='Training Loss')
plt.xlabel('Epochs')
plt.ylabel('Loss')
plt.legend()
```

**Comment by quangchien99** [comment]
So how do i get the accuracy value plot?

**Comment by ptran1203** [comment]
As  I mentioned above, object detection  problem is usually evaluated by  mAP metric not accuracy, I will try to implement it in keras

**Comment by quangchien99** [comment]
Thank u, however, i run your code in google colab, when training - the loss is so high - in epochs 1 is 1.7 , and epochs 30 is 0,7.
Is it ok?

**Comment by ptran1203** [comment]
If the loss is reducing when training, it's ok. You can try to train more epochs or increase the learning rate, eg. 1e-3

**Comment by quangchien99** [comment]
Oke thank u so muchhhh. You're amazing
